### PR TITLE
release: Add blog post and notes for release 0.16.0

### PR DESCRIPTION
**Pull request by razvand**
Add blog post, notes and support files for release 0.16.0 (Telesto).

### CODE REVIEW

File: content/releases/v0.16.0.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+version: 0.16.0
+releaseDate: 2024-01-02
+codename: Pandora
+commit: TODO
+blogPost: /blog/2024-02-02-unikraft-releases-telesto
```
Comment (StefanJum) [inline content/releases/v0.16.0.mdx:6]:
```suggestion
blogPost: /blog/2024-01-02-unikraft-releases-telesto
```

File: content/blog/2024-01-02-unikraft-releases-telesto.mx [hunk 2]
```diff
@@ -0,0 +1,234 @@
+---
```
Comment (StefanJum) [inline content/blog/2024-01-02-unikraft-releases-telesto.mx:1]: Rename the file to `.mdx`

File: content/releases/v0.16.0.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
+---
+version: 0.16.0
+releaseDate: 2024-01-02
+codename: Pandora
```
Comment (StefanJum) [inline content/releases/v0.16.0.mdx:4]:
```suggestion
codename: Telesto
```

**Review by StefanJum — CHANGES_REQUESTED** [review]

**Comment by nderjung** [comment]
Hi, I've been working on edits on the original hackmd doc, I can see these have not yet made its way to the PR yet

**Comment by razvand** [comment]
> Hi, I've been working on edits on the original hackmd doc, I can see these have not yet made its way to the PR yet

I'm now copying the updates to this PR.

**Comment by razvand** [comment]
@nderjung , we won't wait for updates from Andre and Simon to fill TODOs. I will fill the TODO items, and that would be it. The release is already late as it is.

**Comment by razvand** [comment]
> @nderjung , we won't wait for updates from Andre and Simon to fill TODOs. I will fill the TODO items, and that would be it. The release is already late as it is.
> 
> We can do a blog post later on.

Indeed. A blog post is the best way to do it. It's too much "boilerplate" content for a release blog post.

**Review by StefanJum — APPROVED** [review]
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**